ConsentViewController/Classes/SourcePointClient/SimpleClient.swift

```diff
@@ -95,28 +95,23 @@ class SimpleClient: HttpClient {
                 self?.logResponse(urlRequest, data)
 
                 if error != nil {
-                    var spError = GenericNetworkError(request: urlRequest, response: response as? HTTPURLResponse)
                     if let response = response as? HTTPURLResponse {
                         switch response.statusCode {
-                        case 400...499:
-                            spError = ResourceNotFoundError(request: urlRequest, response: response)
-
-                        case 500...599:
-                            spError = InternalServerError(request: urlRequest, response: response)
+                        case 408:
```

Comment:
@andresilveirah attention!

Reply:
@andresilveirah I can make this error be thrown disregard of 408, just every time "something went wrong"

Reply:
@Nevazhnovu makes sense. So we have 3 possible errors being returned here:
1. InvalidResponse
2. Timeout
3. Something wrong

Correct?

Reply:
@andresilveirah that's correct. 
My main concern is:
Should I erase the `switch-case` and `GenericNetworkError` and substitute them with the `ConnectionTimeoutAPIError` throw?
In Android, `TimeoutException` "is thrown to indicate that an input or output data transfer has been terminated because the thread performing it was interrupted" which looks like a generic case, not an exact 408 Server Timeout Error.